### ISSUE TITLE
ColorField Support

### ISSUE THREAD

**Issue by codematsing** [jrief/django-formset#126]
Hi, was wondering if I could use external form libraries together with this library?

https://pypi.org/project/django-colorfield/ doesn't seem to be supported.

Form is rendering properly however saving the form seems as if it does not clean the values inputted in field widget

**Comment by jrief** [comment]
django-colorfield depends on [jscolor](https://www.npmjs.com/package/jscolor), which a) depends on jQuery and b) hasn't been maintained for 7 years. I thus will not support it.

However, it might be that in the near future I will add a widget to edit colors. If you have any modern JS libraries in mind, please let lew know.

**Comment by codematsing** [comment]
Oh I see will look into this and excited for this update as well. Thanks so much!

**Comment by codematsing** [comment]
Closing this. Just use

widgets.Input({"type":"color"})

Just an issue with #61 still occurring - but unrelated to use of color input per se

**Comment by jrief** [comment]
Yes, the built-in color field is quite useful but lacks of a few features other, more specialized libraries can offer. In that sense it's similar to an `<input type="date" />` field which offers a date-picker but which is far from really good implementaions.
